DESCRIPTION

```diff
@@ -32,5 +32,6 @@ Suggests:
     latticeExtra
 LinkingTo: 
     Rcpp (>= 0.12.16),
-    RcppEigen (>= 0.3.3.4.0)
+    RcppEigen (>= 0.3.3.4.0),
+    testthat
```

Comment:
Do we need to link to `testthat`? In R parlance, "LinkingTo" refers to sharing compiled (C / C++) code but I don't see that being used here

Reply:
When I want to add test file for functions from package’s compiled library, it seems that without this link, testtaht::use_catch cannot find <testthat.h>. I will remove this since most functions are in head file.

Reply:
No need to remove it - it seems like a good thing to use. (I just couldn't find the actual use of it since you deleted the tests in subsequent commits on this branch.)

Reply:
So where is this being used? I don't see it in the test files.

Reply:
Hi mentor, it is not being used now, I previously thought I was going to formally test c++ helper function.